src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import DateUtils from '@libs/DateUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+import CardSectionActions from './CardSectionActions';
+import CardSectionDataEmpty from './CardSectionDataEmpty';
+
+function CardSection() {
+    const {translate, preferredLocale} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const [fundList] = useOnyx(ONYXKEYS.FUND_LIST);
+
+    const defaultCard = useMemo(() => Object.values(fundList ?? {}).find((card) => card.isDefault), [fundList]);
+
+    const cardMonth = useMemo(() => DateUtils.getMonthNames(preferredLocale)[(defaultCard?.accountData?.cardMonth ?? 1) - 1], [defaultCard, preferredLocale]);
```

Comment:
Does accountData.cardMonth start at 1?

Reply:
I think so 

@amyevans Could you confirm this?

Reply:
cardMonth starts at 1, yes.

I don't understand the data source though (`.isDefault`, `defaultCard?.accountData?`). Are these just placeholders or is it supposed to be real data at this point? This is what should be used 😄 -
![payment-card-data](https://github.com/Expensify/App/assets/7277067/3687f092-3606-44cc-8203-bd5010347da3)

Reply:
updated

Reply:
@amyevans Is that screenshot up to date? According to onyx types `fundList` is an object, it has `isDefault` field and the objects shown in your screenshot are nested inside `accountData`

Reply:
[Talked about this internally](https://expensify.slack.com/archives/C036QM0SLJK/p1717608307962959), we're not correctly populating the fundList onyx key in the backend. We're working on fixing it. @pasyukevich can you changed this back to what you had before?

Reply:
@blimpich reverted